PowerEditor/installer/nativeLang/zulu.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8" ?>

```

Comment:
Same question as above one.

Reply:
Changed from UTF8 with BOM -> plain UTF8 otherwise the switch to this language is not working.

Reply:
Good job then

Reply:
@chcg could you merge a commit properly?
I can always do it, but if you can, it'll be better.